src/player.cpp

```diff
@@ -629,6 +629,8 @@ player::player() : Character()
 
     morale.reset( new player_morale() );
     last_craft.reset( new craft_command() );
+
+    home = NULL;
```

Comment:
`player::home` is an object of type `base_home`, not a pointer. You can't assign `NULL` to it.

Reply:
ment for `player::home` to be a way to reference base.

Reply:
changed base::home to a pointer